### ISSUE TITLE
Can't open local media

### ISSUE THREAD

**Issue by HakeemOlajuwon** [lrorpilla/jidoujisho#316]
Device - Samsung Galaxy TAB S7+, Android 13

Unable to access the file picker (I think that's what it's called?) on this device.

Here is the most recent log.  Thanks!
[2410202300.log](https://github.com/lrorpilla/jidoujisho/files/13115483/2410202300.log)

**Comment by parkertomf** [comment]
Are you also getting the `the requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired.` Issue?

My book had been in there for months no problem, then randomly disappeared, and now I can't add it back in without that failure anymore.

**Comment by HakeemOlajuwon** [comment]
Nope,  I'm not getting any message at all when trying to navigate to the Local Media in the "Player" section.  

However, I am able to navigate to books in the "Reader" section, if that helps diagnose the issue.
